aws/data_source_aws_cloudfront_distribution.go

```diff
@@ -0,0 +1,66 @@
+package aws
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func dataSourceAwsCloudFrontDistribution() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceAwsCloudFrontDistributionRead,
+
+		Schema: map[string]*schema.Schema{
+			"id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"active_trusted_signers": {
+				Type:     schema.TypeMap,
+				Computed: true,
+			},
```

Comment:
I would suggest removing `active_trusted_signers` for now from the schema here, any handling in the Read function, and the associated documentation. Its implementation (even in the resource) is suspect. See also: https://github.com/terraform-providers/terraform-provider-aws/issues/8902